### PR TITLE
Update ThemingController.php

### PR DESCRIPTION
**Pull request by rakekniven**
Changed wording according to discussion at Transifex. See https://www.transifex.com/nextcloud/nextcloud/translate/#de/$/121491259?issue=yes
Aligned ending of error messages

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

File: apps/theming/lib/Controller/ThemingController.php [hunk 1]
```diff
@@ -218,14 +218,14 @@ public function updateLogo() {
...
-			UPLOAD_ERR_OK => $this->l10n->t('There is no error, the file uploaded with success'),
+			UPLOAD_ERR_OK => $this->l10n->t('The file was uploaded'),
 			UPLOAD_ERR_INI_SIZE => $this->l10n->t('The uploaded file exceeds the upload_max_filesize directive in php.ini'),
 			UPLOAD_ERR_FORM_SIZE => $this->l10n->t('The uploaded file exceeds the MAX_FILE_SIZE directive that was specified in the HTML form'),
 			UPLOAD_ERR_PARTIAL => $this->l10n->t('The file was only partially uploaded'),
 			UPLOAD_ERR_NO_FILE => $this->l10n->t('No file was uploaded'),
 			UPLOAD_ERR_NO_TMP_DIR => $this->l10n->t('Missing a temporary folder'),
-			UPLOAD_ERR_CANT_WRITE => $this->l10n->t('Failed to write file to disk.'),
-			UPLOAD_ERR_EXTENSION => $this->l10n->t('A PHP extension stopped the file upload.'),
+			UPLOAD_ERR_CANT_WRITE => $this->l10n->t('Failed to write file to disk'),
```
Comment (comradekingu) [inline apps/theming/lib/Controller/ThemingController.php]: The most commonly used convention in NC is:
"Could not write file to disk"

Comment (juliushaertl) [reply apps/theming/lib/Controller/ThemingController.php]: @rakekniven Can you adjust that as well?

**Comment by nickvergessen** [comment]
Test fails:
> OCA\Theming\Tests\Controller\ThemingControllerTest

**Review by nickvergessen — CHANGES_REQUESTED** [review]
Tests

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**